<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_Main</class>
 <widget class="QMainWindow" name="window_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelHead">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>691</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Заполните поля формы</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelFamilia">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Фамилия</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPatronimic">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Отчество</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBirthDay">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата рождения</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAny">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>181</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Дополнительная информация</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateBirthDay">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineFamilia">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineName">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePatronimic">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textAny">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>240</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupGender">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>Пол</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="radioFemina">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>82</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ж</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioMale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>М</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupMatrimony">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>240</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Семейный статус</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="radioMarred">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>женат/замужем</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioFree">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>холост/незамужем</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonSave">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
